scripts/buildRpm.sh

```diff
@@ -3,12 +3,22 @@ set -e
 
 PACKAGE=MurMurHash
 
-if [[ $# -lt 1 ]] ; then
-    echo 'Usage:  sh buildRpm <BUILD_NUMBER>'
-    exit 0
+if [[ $# -ne 2 ]] ; then
+   echo 'Usage:  sh buildRpm <BUILD_TYPE> <BUILD_NUMBER>'
```

Comment:
Please switch the order of these input arguments. Our typical format is `<BRANCH> <BUILD-NUMBER> <BUILD-TYPE>`. If we don't have BRANCH, the order should still follow.